daug/cogs/utils.py

```diff
@@ -7,9 +9,46 @@ class Utils(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
         self.id = self.bot.config['Daug']['guild_id']
+        self.committer_role_id = self.bot.config['Daug']['role_committer_id']
+        self.staff_role_id = self.bot.config['Daug']['role_staff_id']
+        self.committer_perm_role_id = self.bot.config['Daug']['role_committer_perm_id']
+        self.staff_perm_role_id = self.bot.config['Daug']['role_staff_perm_id']
+
+    def is_committer(self, author) -> bool:
+        if self.committer_role_id not in [role.id for role in author.roles]:
+            return False
+        return True
+
+    def is_staff(self, author) -> bool:
+        if self.staff_role_id not in [role.id for role in author.roles]:
+            return False
+        return True
```

Comment:
```suggestion
    def is_committer(self, author) -> bool:
        return self.committer_role_id in [role.id for role in author.roles]

    def is_staff(self, author) -> bool:
        return self.staff_role_id in [role.id for role in author.roles]
```

好みもありそうだけど、こっちの方がシンプルで読みやすい気がする